### PR TITLE
docs(installation): add DDS setting to additional-settings-for-developpers

### PR DESCRIPTION
**Pull request by takam5f2**
Signed-off-by: Takayuki AKAMINE <takayuki.akamine@tier4.jp>

## Description

I would like to add DDS settings for beginners to execute Autoware as they expect. They tend to execute Autoware without taking care of DDS settings. It causes performance problem which cannot seen on Rviz.

Some of beginners reported that sensing pre-processing runs at less than 10 Hz, it is not expected (https://github.com/tier4/CARET_analyze/issues/155#issuecomment-1275625569). One of them gave me a figure which shows  this unexpected behavior. It is a time-series graph plotting frequency of callback execution in `/sensing/lidar/top/velodyne_convert_node`. The frequency decreased time by time and below 10 Hz.

image (1)

Tackling this issue with him, I found that default receive buffer size is so small that Autoware cannot receive large data like camera image or LiDAR point cloud. Most of beginners must not be concerned with it.

I think that DDS setting should be more common topic for Autoware users, so this motivated me to issue the PR.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by takam5f2** [comment]
@kenji-miyake  
I assigned you as a reviewer of this PR because this PR will updates the page you created. If you think that another reviewer is appropriate, please assign him or her.

**Comment by shmpwk** [comment]
There are related document [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/support/troubleshooting.md#runtime-issues). I wonder it can be merged or refer each other.

**Review by kenji-miyake — COMMENTED** [review]
I think it's good to add this content to this page.
Regarding native-check and DDS-specific issues, it would be good to get reviews from someone else as well.

@xmfcx Would it be possible for you to assign a developer?

**Comment by kenji-miyake** [comment]
> There are related document here. I wonder it can be merged or refer each other.

My proposal is to write general settings on this page, and link to the troubleshooting page for issues that many people will encounter.

File: docs/installation/additional-settings-for-developers/index.md [hunk 1]
```diff
@@ -45,3 +45,39 @@ export ROS_DOMAIN_ID=X
...
+```bash
+export CYCLONEDDS_URI=file:///absolute/path/to/cyclonedds_config.xml
+sudo sysctl -w net.core.rmem_max=2147483647
+```
+
+Refer to [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html) for more information. Reading user guide for chosen DDS is helpful for more understanding.
+
+### Tuning DDS for multiple host computers (for advanced users)
+
+When Autoware runs on multiple host computers, IP Fragmentation should be taken account of. As [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html#cross-vendor-tuning) recommends, parameters for IP Fragmentation should be given as well as the following example.
```
Comment (xmfcx) [inline docs/installation/additional-settings-for-developers/index.md:78]:
```suggestion
When Autoware runs on multiple host computers, IP Fragmentation should be taken into account. As [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html#cross-vendor-tuning) recommends, parameters for IP Fragmentation should be set as shown in the following example.
```
nit: grammar

**Review by kaancolak — APPROVED** [review]
LGTM

I think it makes sense to add default settings. It directly affects overall Autoware performance.

File: docs/installation/additional-settings-for-developers/index.md (hunk 1 above)
Comment (takam5f2) [reply docs/installation/additional-settings-for-developers/index.md:78]: @xmfcx  Thank you, and I do fix the document as you suggested.

**Comment by takam5f2** [comment]
@kaancolak  Thank you for reviewing.

**Pull request merged**